demos/simple-storage/repro-build-protocol/web3api.build.yaml

```diff
@@ -0,0 +1,9 @@
+image: 
+  name: build-env
+  dockerfile: ./myCustomDockerfile #Comment to use default image
+env:
```

Comment:
I feel like something other than `env` should be used here as I feel like that's more typically for environment variables. Maybe `build` since you're defining the various files/directories for the build process and that can be expanded on later? Or maybe `source` and then the child properties can be `input`, `output` and eventually all support files/folders/globs?